andela/ah-codeblooded-frontend
# #160577585  Enable Users to opt in and out of notifications

## What does this PR do?
This PR allow users to subscribe and unsubscribe to email notifications

## Description of Task to be completed?

### Why this is important
Notifications give users to opportunity to know what they may have missed which may be important to them. However, forcing notifications on users may be seen as annoying. Users should have the ability to opt in and out of (or turn on and off) notifications.

### As a User
I want to be able to opt in and out of (or turn on and off) notifications.

### How should this be manually tested?**

- Log in to the app.
- Navigate to /settings route on the Navbar
- You should be taken to a page that displays the subscription switch

### Screenshots (if appropriate)

#### When the user subscribes to email notifications
![image](https://user-images.githubusercontent.com/31212441/48904416-546cd800-ee6f-11e8-8771-b30fd5115671.png)


#### When the user is unsubscribes from email notifications
![image](https://user-images.githubusercontent.com/31212441/48904452-736b6a00-ee6f-11e8-9e85-bb3d3ca8c6eb.png)





## What are the relevant pivotal tracker stories?
The relevant pivotal story is [here](https://www.pivotaltracker.com/n/projects/2198535/stories/160577585)
